app/templates/components/lg-image-gallery.hbs

```diff
@@ -0,0 +1,25 @@
+{{#each imageSet.images as |image|}}
+  <span style='float: left'>
+    <img {{bind-attr src=image.url}} width='100' height='100'>
+    <ul>
+      {{#if editingEnabled}}
+        <li>{{view 'select' content=imageTypes value=image.imageType}}</li>
+        <li>{{input type='checkbox' checked=image.isPublic}} Public</li>
```

Comment:
nvm, I get it. The details for all the images are rolled up and saved at one go, later.